Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Options.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Represents the control that redistributes space between columns or rows of a Grid control.
+    /// </summary>
+    public partial class GridSplitter
+    {
+        /// <summary>
+        /// Identifies the <see cref="ResizeDirection"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty ResizeDirectionProperty
+            = DependencyProperty.Register(
+                nameof(ResizeDirection),
+                typeof(GridResizeDirection),
+                typeof(GridSplitter),
+                new PropertyMetadata(GridResizeDirection.Auto, OnResizeDirectionChange));
+
+        /// <summary>
+        /// Identifies the <see cref="ResizeBehavior"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty ResizeBehaviorProperty
+            = DependencyProperty.Register(
+                nameof(ResizeBehavior),
+                typeof(GridResizeBehavior),
+                typeof(GridSplitter),
+                new PropertyMetadata(GridResizeBehavior.BasedOnAlignment, OnResizeBehaviorChange));
+
+        /// <summary>
+        /// Identifies the <see cref="VerticalIconText"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty VerticalIconTextProperty
+            = DependencyProperty.Register(
+                nameof(ResizeBehavior),
+                typeof(string),
+                typeof(GridSplitter),
+                new PropertyMetadata(GripperBarVertical));
+
+        /// <summary>
+        /// Identifies the <see cref="HorizontalIconText"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty HorizontalIconTextProperty
+            = DependencyProperty.Register(
+                nameof(HorizontalIconText),
+                typeof(string),
+                typeof(GridSplitter),
+                new PropertyMetadata(GripperBarHorizontal));
```

Comment:
If we can't go the Content route as mentioned, I think it would be better for this to be object, and bound to a ContentControl, so you can also choose to insert image, background gradients for a rounded look or anything else you want and not just a text string.